api/src/main/resources/workbench.yaml

```diff
@@ -538,6 +538,32 @@ paths:
 
   # Notebook clusters ####################################################################
 
+  /v1/admin/security/clusters/{billingProjectId}/delete-clusters:
```

Comment:
[Important] Per discussion on Slack, we'll want to flex the requirements to this ticket a bit and add support for deleting a specific named cluster from within the project. The easiest way to handle this would probably be to add an optional body parameter to this API method, "cluster_names", which allows one or more cluster names to be provided.

The idea is that if / when we implement auto-cluster-delete in response to high-egress events, we'll likely have the specific name of the offending cluster. To reduce the blast radius of the alert response, we'll favor deleting just the known cluster name rather than blasting all clusters from the project.

Do you think it'll be possible to fit that into this PR without too much re-working? It could also be a follow-up – your judgement call to make.

Reply:
Probably, let me add that.